src/main/java/com/conveyal/datatools/manager/models/FeedSource.java

```diff
@@ -615,7 +615,7 @@ public FeedSource clone () throws CloneNotSupportedException {
      * occurring at the same time).
      */
     public boolean hasJobsInProgress() {
-        return JobUtils.filterActiveJobs(JobUtils.getAllJobs()).stream().anyMatch(job -> {
+        return JobUtils.filterStagedAndActiveJobs(JobUtils.getAllJobs()).stream().anyMatch(job -> {
```

Comment:
This creates a regression from https://github.com/ibi-group/datatools-server/commit/db462d05dd7622ef6f0ff03769350ce2e9622347, which I suppose created the other regression that this PR aims to fix. This changed filtering of jobs now filters out actively running recurring jobs which have already "completed" at least once. I added a comment about this, but it's easy to miss this nuance (see below) https://github.com/ibi-group/datatools-server/blob/ddb3cc913525fe93598fdd870b86fded5127f995/src/main/java/com/conveyal/datatools/common/status/MonitorableJob.java#L33-L37

Therefore, I think if we change this to the following, it should work again for auto-deployment purposes.

```suggestion
        return JobUtils.getAllActiveJobs().stream().anyMatch(job -> {
```